mist-lib/src/services/SignalTransductionService.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const csv = require('csvtojson'),
+	path = require('path')
+
+const DefaultTableFilePath = path.resolve(__dirname, '..', 'signal-transduction', 'mist3-sig-trans-2017.csv')
+
+
+module.exports =
+class SignalTransductionService {
+	constructor(tableFilePath) {
+		this.tableFilePath = tableFilePath ? tableFilePath : DefaultTableFilePath
+		this.pfql = []
+		this.sigData = []
+	}
+
+	parseTable_() {
+		return new Promise((res, rej) => {
```

Comment:
Avoid abbreviations. Rename `res` and `rej` to `resolve` and `reject`, respectively.

Reply:
Done